src/scanoss/cli.py

```
@@ -323,9 +323,10 @@ def setup_args() -> None:  # noqa: PLR0912, PLR0915
     c_provenance = comp_sub.add_parser(
         'provenance',
         aliases=['prov', 'prv'],
-        description=f'Show Provenance findings: {__version__}',
-        help='Retrieve provenance for the given components',
+        description=f'Show GEO Provenance findings: {__version__}',
+        help='Retrieve geoprovenance for the given components',
     )
+    c_provenance.add_argument('--origin', action='store_true', help='Retrieve geoprovenance using contributors origin (default: declared origin)')
```

Ruff (0.8.2)

Line too long (146 > 120)

(E501)